core/src/test/java/apoc/periodic/PeriodicTest.java

```diff
@@ -278,6 +279,27 @@ public void testIterate() throws Exception {
                 row -> assertEquals(100L, row.get("count"))
         );
     }
+    
+    @Test
+    public void testStatusDetailsPeriodicIterate() {
+        db.executeTransactionally("UNWIND range(1,9999) AS x CREATE (:StatusIterate)");
+        KernelTestUtils.checkStatusDetails(db, 
+                "CALL apoc.periodic.iterate('match (p:StatusIterate) return p', 'SET p.lastname =p.name REMOVE p.name', {batchSize:10,parallel:true})", 
+                Collections.emptyMap(), 
```

Comment:
Why is this status empty?

Reply:
This `Collections.emptyMap()` is just the `params` that I pass in the `db,executeTransactionally(query, params)`